ODataConnectedService/src/ODataConnectedService.csproj

```diff
@@ -34,13 +39,13 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.OData.ConnectedService</RootNamespace>
     <AssemblyName>Microsoft.OData.ConnectedService</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Was there a reason to move to NET Framework 4.6? Are we able to stick with 4.5 to minimize changes?